
foreign_ip: 192.168.5.24
foreign_port: 11311

connectCheck_Hz: 500
msgsFrequency_Hz: 20

pub_queue_size: 1
pub_name: master_bridge_motor
 
TopicList: [xycar_motor]

